Update README.md (#74)
cedar-policy/cedar-spec: cedar-policy-generators/README.md

# Cedar Policy Generators

> This folder contains code for DRT input (i.e., schemas, entities, policies, requests, and etc.) generation using the [arbitrary](https://docs.rs/arbitrary/latest/arbitrary/index.html) crate.

Reviewer: ```suggestion
This folder contains code for DRT input (i.e., schemas, entities, policies, requests, etc.) generation using the [arbitrary](https://docs.rs/arbitrary/latest/arbitrary/index.html) crate.
```

> Currently the only user of this crate is [cedar-drt](../cedar-drt).
> An example is the target [`abac-type-directed`](../cedar-drt/fuzz/fuzz_targets/abac-type-directed.rs#L58-L72) that generates arbitrary typed ABAC policies.

Reviewer: ```suggestion
Currently the only user of this crate is [cedar-drt](../cedar-drt).
For example, see the generator for the target [`abac-type-directed`](../cedar-drt/fuzz/fuzz_targets/abac-type-directed.rs#L58-L72).
```

> We plan to extend the generators to take inputs as custom schemas.

Reviewer: Delete "We plan to extend...". This will become outdated soon due to PR #73.
